src/main/java/core/basesyntax/strategy/impl/BalanceTypeStrategy.java

```diff
@@ -0,0 +1,16 @@
+package core.basesyntax.strategy.impl;
+
+import core.basesyntax.db.Storage;
+import core.basesyntax.model.FruitTransaction;
+import core.basesyntax.strategy.TypeStrategy;
+
+public class BalanceTypeStrategy implements TypeStrategy {
```

Comment:
```suggestion
public class BalanceHandler implements OperationHandler {
```
and rename other handler appropriately